managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -248,6 +248,7 @@ public void operationComplete(ManagedLedgerInfo mlInfo, Stat stat) {
                     if (log.isDebugEnabled()) {
                         log.debug("[{}] Opening legder {}", name, id);
                     }
+                    store.recordRead();
```

Comment:
yes, have thought about it (read will be quick as coming from memory directly) but then concluded to keep one by considering that we are anyway having mean, percentile : 95, 99, 99.9, 99.99 so, write will be automatically reflected as part of 99.9 or 99.99. So, avoided having additional histogram.
But will introduce separate histogram.